src/storage/collections.js

```diff
@@ -11,5 +11,6 @@ module.exports = {
   MESSAGES: 'messages',
   REQUESTS: 'requests',
   NOTIFICATIONS: 'notifications',
-  CONFIG: 'config'
+  CONFIG: 'config',
+  LIBRARIES: 'libraries'
```

Comment:
Just to clarify this is the first _fhir_ collection which isn't resourceType + s. The places this happen are:

- `routes/index` `/trigger` POST handler: this shouldn't matter though because the resource here is from a trigger and is thus not a Library
- `services/subscription` `handleReportTriggerResource`: this shouldn't matter either because the resource here is also from a trigger and is thus not a Library
- `utils/actions` `extract-research-data`: this could _potentially_ cause an issue if for some reason a Library was requested as part of report
- `utils/fhir` `getResources`: this is probably the only real potential issue. If the Library is returned back with the request for the PlanDefinition then this code could be run for a Library resource.

Reply:
I did not really think about the consequences when I added this collection. Should I make it `Librarys` to remain consistent?

Reply:
Sounds good. There are some other options, but all have their own downsides, so I think this is probably the best. 

- we could make the constant LIBRARIES but make the collection name librarys. This could get confusing if you try to go directly into the DB and can't find "libraries"
- we could make a more complex "pluralize" function, or import a library. Probably overkill
- we could patch the english language but i think that PR is unlikely to be merged